src/components/studies/app-design/AppDesign.tsx

```diff
@@ -316,6 +321,80 @@ const AppDesign: React.FunctionComponent<
     isIrbEmailValid: true,
   })
 
+  const [errorState, setErrorState] = React.useState({
```

Comment:
you can have a type for errorState where all those props are optional. This way you can just initially set it to the empty object and don't need the lengthy set statements